src/commons/utils/date.ts

```diff
@@ -0,0 +1,26 @@
+import { EventDuration } from "@/apollo/types";
+import { format } from "date-fns";
+
+export const getDisplayDate = (durations?: EventDuration[]) => {
+  if (!durations) {
+    return "-";
+  }
+  const startDateText = format(new Date(durations[0].start), "ccc, d MMM");
+  if (durations.length !== 0) {
```

Comment:
Why is this if here? Suppose durations can be blank list, then [] will never return '-' from above condition (if !durations). And if durations is actually blank list, code will break at line before because you already accessed position 0 to get startDateText.

Reply:
This is intentional krub. I have consulted with UXUI team